README.md

```
@@ -69,6 +69,36 @@ All of the code sits here:
 
 <img src="./public/screenshots/demo-admin-view.png/"/>
 
+## Add to website
+
+<p align="center">
+  <img src="./public/screenshots/demo-widget.png" style="background-color: white" />
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)